### DISCUSSION TITLE
How install watcom only by command line without the interface?

### DISCUSSION THREAD

**Discussion by DevKiDCosmo** [open-watcom/open-watcom-v2#1296] [Q&A] [answered]
## How install watcom only by command line without the interface?

I create a repository that uses Watcom. I want to create **CI** and I need to find a way to install Watcom without human interference.

**Comment by jmalak** [comment, marked as answer]
There is [project of OW installation](https://github.com/open-watcom/setup-watcom) for such purpose, that you can simply install OW for CI. You can select version which you need.
Anyway you can install OW simply by downloading OW binary snapshot and [setup OW](https://github.com/open-watcom/open-watcom-v2/wiki/OW-tools-usage-Overview)

**Reply by DevKiDCosmo** [reply]
Thank you
